docs/content/setup/_index.md

```diff
@@ -9,18 +9,19 @@ tags = ["setup"]
 
 # Setting up Infection Monkey
 
-Setting up the Infection Monkey is easy! First, you need to {{% button href="https://infectionmonkey.com/" icon="fas fa-download" %}}Download the Infection Monkey{{% /button %}}.
-
+Setting up the Infection Monkey is easy! First, you need to download the Infection Monkey.
 Once you've downloaded an installer, follow the relevant guide for your environment:
 
-{{% children %}}
-
-After setting the Monkey up, check out our [Getting Started](/usage/getting-started) guide!
+- [Windows](https://techdocs.akamai.com/infection-monkey/docs/windows)
+- [Docker](https://techdocs.akamai.com/infection-monkey/docs/docker)
+- [Linux](https://techdocs.akamai.com/infection-monkey/docs/linux)
+- [AWS](https://techdocs.akamai.com/infection-monkey/docs/aws)
+- [Azure](https://techdocs.akamai.com/infection-monkey/docs/azure)
+- [Accounts and security](https://techdocs.akamai.com/infection-monkey/docs/accounts-and-security)
 
-{{% notice tip %}}
-You can find information about [operating system compatibility and support here](../reference/operating_systems_support).
-{{% /notice %}}
+After setting the Monkey up, check out our [Getting started guide](doc:getting-started).
```

Comment:
Similarly, this link doesn't work in our development platform, which is problematic for us. Is there a specific tool we should be using to render the markdown during development?